detect-nuget-inspector/detect-nuget-inspector/DependencyResolution/Project/ProjectReferenceResolver.cs

```diff
@@ -45,8 +48,13 @@ public DependencyResult Process()
                     
                     var versionMetaData = reference.Metadata.Where(meta => meta.Name == "Version").FirstOrDefault();
                     var versionOverrideMetaData = reference.Metadata.Where(meta => meta.Name == "VersionOverride").FirstOrDefault();
+                    var privateAssetsMetaData = reference.Metadata.Where(meta => meta.Name == "PrivateAssets").FirstOrDefault();
+
+                    bool isDevDependency =
```

Comment:
I found this variable name a bit misleading. The dependency can be a dev dependency, yet if it is not being excluded this variable will suggest that it is not a dev dependency.
It may be clearer to separate it into two variables: `isDevDependency` and `excludeDev`, then check for both below.

Reply:
Yeah certainly, I would change the name for this variable in all the places to make it more meaningful.

Reply:
One more question: on line 76 addNugetDependency() can be called even if this variable is true. Is that correct?

Reply:
No it is not, will change that. Thanks